circle.yml

```diff
@@ -283,7 +283,7 @@ commands:
       - run:
           name: Prepare macOS
           command: |
-            brew install cmake ccache glfw ninja pkgconfig qt chargepoint/xcparse/xcparse
+            brew install cmake ccache glfw ninja pkgconfig qt chargepoint/xcparse/xcparse libuv
```

Comment:
FYI, we do not use Circle CI anymore, this file should be removed at some point.

Reply:
Yup, for whatever reason I thought otherwise, and took a sec to realize we're using GH pipelines here. Will revert changes to this file 👍